doc/api/modules.md

```diff
@@ -850,6 +850,21 @@ by the [module wrapper][]. To access it, require the `Module` module:
 const builtin = require('module').builtinModules;
 ```
 
+### module.makeRequire(filename)
+
+* `filename` {string|[URL][]}
+* Returns: {[require][]} Require function
+
+```js
+// create a require function which will resolve relative
+// to the current ES Module file
+const require = module.makeRequire(new URL(import.meta.url));
+```
+
+<!-- YAML
+added: REPLACEME
+-->
```

Comment:
This should go directly below the heading

Reply:
lol i have no idea how this got down here